yb-voyager/src/srcdb/mysql_test.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package srcdb
+
+import (
+	"testing"
+
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils/sqlname"
+	testutils "github.com/yugabyte/yb-voyager/yb-voyager/test/utils"
+	"gotest.tools/assert"
+)
+
+func TestMysqlGetAllTableNames(t *testing.T) {
+	sqlname.SourceDBType = "mysql"
+
+	// Test GetAllTableNames
+	actualTables := testMySQLSource.DB().GetAllTableNames()
+	expectedTables := []*sqlname.SourceName{
```

Comment:
Reading this test-case in isolation, it's confusing where these tables come from. I would be in favour of NOT having a default schema that is assumed with testcontainer startup. Don't really see a point of it. 
I think it would be better for each testcase to be self-contained. clearly set up it's schema objects at the beginning of the test, run its tests, delete/drop the schema objects. 

Another reason to avoid a single common schema is that over time, we will end up writing all the DDLs for our tests into that schema file itself, which beats the point of having self-container "unit" tests.

Reply:
Done @makalaaneesh

Reply:
Nice! `ExecuteSqls` is nice, and using it with defer for cleanup is very useful